tests/templates/pr-create.md.ejs.js

```diff
@@ -29,127 +29,120 @@ describe("pr-create.md.ejs", () => {
-                            message: "Foo bar"
-                        }
+                pull_request: {
+                    title: "Fix: foo (fixes #9012)"
+                },
+                changes: {
+                    title: {
+                        from: "Fix foo"
                     }
-                ]
+                }
             }
         });
 
-        assert.ok(result.indexOf("begin with a tag") > -1);
+        assert.equal(result.trim(), "LGTM");
     });
 
-    it("should mention commit message length when there's a message longer than 72 characters", () => {
+    it("should not leave a response when the pull request title stays the same", () => {
         const result = ejs.render(TEMPLATE_TEXT, {
             payload: {
+                action: "edited",
                 sender: {
                     login: "nzakas"
                 },
-                commits: 1
-            },
-            meta: {
-                cla: true,
-                commits: [
-                    {
-                        commit: {
-                            message: "Fix:56789012345678901234567890123456789012345678901234567890(fixes #9012)"
-                        }
+                pull_request: {
+                    title: "Fix: foo (fixes #9012)"
+                },
+                changes: {
+                    title: {
+                        from: "Fix: foo (fixes #9012)"
                     }
-                ]
+                }
             }
         });
 
-        assert.ok(result.indexOf("72 characters") > -1);
+        assert.strictEqual(result.trim(), "");
     });
 
-    it("should not mention commit message length when there's a multi-line message with first line not over 72 characters", () => {
+    it("should not leave a response when new commits are pushed to the pull request", () => {
         const result = ejs.render(TEMPLATE_TEXT, {
             payload: {
+                action: "synchronized",
                 sender: {
                     login: "nzakas"
                 },
-                commits: 1
-            },
-            meta: {
-                cla: true,
-                commits: [
-                    {
-                        commit: {
-                            message: "Fix:56789012345678901234567890123456789012345678901234567890(fixes #901)\r\n1234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890"
-                        }
-                    }
-                ]
+                pull_request: {
+                    title: "Fix: foo (fixes #9012)"
+                }
             }
         });
 
-        assert.equal(result.trim(), "LGTM");
+        assert.strictEqual(result.trim(), "");
     });
 
-    it("should not mention missing issue when there's one documentation commit", () => {
+    it("should mention commit message format when there's one commit and an invalid commit message is found", () => {
```

Comment:
Yes, the bot would now check the title of the PR rather than the commit messages, so `commits: 1` would not be relevant.